src/cluster/cluster.h

```diff
@@ -0,0 +1,26 @@
+/****
+DIAMOND protein aligner
+Copyright (C) 2013-2018 Benjamin Buchfink <buchfink@gmail.com>
```

Comment:
For files that contain mostly your own code, please put an appropriate copyright notice here, with you and/or your employer as the copyright holder.

Reply:
Will do, but what about files where we have both modified the code?

Reply:
You should amend the copyright notice then, although for files that you only changed minimally you can also leave it, if that's ok with you.

Reply:
For now I have chosen to keep the header as-is and replace the years, name and email address with my (private) credentials. I will check back with Q, what they would prefer, but for the current work I think it would also be appropriate to put my own creds since I did this in my own time.
Do you otherwise agree on that I keep the header "DIAMOND protein aligner"? I think that would be reasonable, since the code is an integral part of DIAMOND and cannot be used as a standalone tool.

Reply:
Yes, I think that's fine.

Reply:
I have updated the headers according to the requirements of my employer and I have not put any shared copyright anywhere.